### PR TITLE
Makes reports submittable, with newly added fields on CDC Report

### LINKED ISSUE #167: Add success alert after submitting report
**Story**: As a provider, I want to be explicitly updated about the status of my report submissions

**Acceptance criteria**:
- success notification (like below) is displayed when report is successfully submitted
- a notification component is created, and used in `Report` component to achieve this

Screenshot from 2019-10-28 12-39-10

### PR DESCRIPTION
**Pull request by jlhogan**
Also adds some new fields to site

NOTE: theres a diff btwn this and invision, where accredited checkbox is below the utilization table. easier and cleaner to impl as part of the form fragment component that way, but if there's a hard need for it to be above we can do that.

see it live here: https://3.136.37.61:5001/ !!!!!!!!!!!!!!!!!

closes #167

### CODE REVIEW

File: src/Hedwig/ClientApp/src/swagger.json [hunk 1]
```diff
@@ -934,32 +960,68 @@
...
                     "$ref": "#/components/schemas/Site"
                   }
+=======
+                  "$ref": "#/components/schemas/CdcReport"
+>>>>>>> [WIP] submit roster with new fields
                 }
               },
               "text/json": {
                 "schema": {
+<<<<<<< HEAD
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/swagger.json]: Eep 👀

File: src/Hedwig/Controllers/ReportsController.cs [hunk 2]
```diff
@@ -46,5 +46,30 @@ int orgId
...
+        if(report.SubmittedAt == null) report.SubmittedAt = DateTime.UtcNow;
+        _reports.UpdateReport(report);
+        await _reports.SaveChangesAsync();
+      }
+      catch (DbUpdateConcurrencyException)
+      {
+        return NotFound();
+      }
+
+      return Ok(report);
```
Comment (cailyncodes) [inline src/Hedwig/Controllers/ReportsController.cs]: Is wrapping this in `Ok` necessary? I ask this as a very general question - I see some with it, some without

Comment (jlhogan) [reply src/Hedwig/Controllers/ReportsController.cs]: does not change the actual content of the response, but does change the type which makes testing cleaner.

File: src/Hedwig/ClientApp/src/swagger.json (hunk 1 above)
Comment (jlhogan) [reply src/Hedwig/ClientApp/src/swagger.json]: blarg

File: src/Hedwig/Models/Region.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+namespace Hedwig.Models
+{
+  public enum Region
+  {
+    Default = 0
```
Comment (cmgiven) [inline src/Hedwig/Models/Region.cs]: The regions are E, NC, NW, SC, SW. Also here: https://github.com/ctoec/ecis-experimental/blob/master/src/Hedwig/ClientApp/src/containers/ReportDetail/CdcRates.ts#L3

Should not block merging this in.

**Pull request merged**